src/main/scala/edg_ide/runner/DesignTopRunConfigurationType.scala

```diff
@@ -82,28 +94,40 @@ class DesignTopRunConfiguration(project: Project, factory: ConfigurationFactory,
   val kFieldDesignName = "DESIGN_NAME"
   val kFieldNetlistName = "NETLIST_NAME"
   val kPdfFileName = "PDF_NAME"
+  val kFieldValueMode = "VALUEMODE_NAME"
```

Comment:
```suggestion
  val kFieldRefdesMode = "REFDESMODE_NAME"
```
Consider making this consistent - this significantly helps readability where similar concepts are named the same.